Examples/MAX78000/CNN/digit-detection-demo/TFT/fthr/bmp/bmp2c.py

```diff
@@ -1,11 +1,35 @@
-###################################################################################################
-#
-# Copyright (C) 2020-2021 Maxim Integrated Products, Inc. All Rights Reserved.
-#
-# Maxim Integrated Products, Inc. Default Copyright Notice:
-# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
-#
-###################################################################################################
```

Comment:
Hope replacing this to the actual "free of charge" version of the copyright notice is fine.